INSTALL.md

```diff
@@ -558,28 +558,27 @@ This is a problem if you are running a keyboard driver for some non-english
 languages. These drivers catch the ^ character to produce the French circumflex
 accent and do not pass it properly to GAP. For macOS users, as a workaround
 please refer to the section "GAP for macOS" below for information on
-how to install readline and section 5 on how to recompile GAP, for windows no
+how to install readline and section 5 on how to recompile GAP, for Windows no
 fix is known. (One can type `POW(a,b)` for `a^b`.)
 
 ## Problems specific to Windows
 
-### Cut and Paste does not work
+The best way to use GAP in Windows is to use the "Windows Subsystem for Linux"
```

Comment:
Elsewhere in this file we write "If you are using the `.exe` installer (which we strongly recommend),"

Which sounds as if it contradicts this. Well, it doesn't really (that sentence was meant to be in contrast to using the `.zip` file version, which we don't provide anymore I think), but a fresh reader can't know that. I guess we should adjust that other place then perhaps?